backend/src/Designer/Controllers/DeploymentsController.cs

```diff
@@ -67,18 +67,21 @@ public async Task<SearchResults<DeploymentEntity>> Get(string org, string app, [
         /// <returns>List of environment names</returns>
         [HttpGet]
         [Route("permissions")]
-        public async Task<List<string>> Permissions([FromRoute] string org)
+        public async Task<ActionResult<List<string>>> Permissions([FromRoute] string org)
         {
-            List<string> permittedEnvironments;
+            List<string> permittedEnvironments = new ();
 
             List<Team> teams = await _giteaService.GetTeams();
-            permittedEnvironments = teams.Where(t =>
-            t.Organization.Username.Equals(org, StringComparison.OrdinalIgnoreCase)
-            && t.Name.StartsWith("Deploy-", StringComparison.OrdinalIgnoreCase))
-                .Select(t => t.Name.Split('-')[1])
-                .ToList();
+            if (teams.Count > 0)
```

Comment:
Is this check necessary? Looks like linq query will work fine with empty collection